.github/workflows/cypress-testing.yml

```diff
@@ -1,8 +1,9 @@
 name: Run Cypress testing suite
 on:
   workflow_dispatch:
-  pull_request:
-    types: [opened, synchronize]
+  push:
+    branches:
+      - development
```

Comment:
I guess this is for rate limiting or something?

Reply:
Not at all, just habit. It can be triggered on every branch, will change